src/types/contentful.ts

```diff
@@ -0,0 +1,20 @@
+export const CONTENTFUL_INCLUDE = {
```

Comment:
Maybe rename this to something like:

`GENRE_CONTENTFUL_INCLUDE` ?  as this is specific for the Genre model

Reply:
I wanted to keep this more generic for everyone to use it.
This way we can keep the same value with the correct description, like below:
```typescript
export const CONTENTFUL_INCLUDE = {
	noInclude: 0, // this adds at least the linked IDs
	movies: 1,
	moviesWithGenre: 2,
        genres: 1,
        genresWithMovie: 2,
} as const;
```